.github/workflows/master.yml

```diff
@@ -23,21 +23,21 @@ jobs:
             spark-hadoop: '2.7'
           - java: 8
             spark: '3.1'
+            spark-hadoop: '3.2'
+          - java: 8
+            spark: '3.2'
```

Comment:
`spark-hadoop: '2.7'` means breaking hadoop3.2?

Reply:
I don't get your ponit.

Previously, we use `kyuubi-spark-sql-engine.jar` build against Spark 3.1 do corss test on spark-3.0 & hadoop-2.7 and spark 3.2 & hadoop 3.2.

Now, we use `kyuubi-spark-sql-engine.jar` build against Spark 3.2 do corss test on spark-3.0 & hadoop-2.7 and spark 3.1 & hadoop 3.2.

Reply:
I mean cross check shall use default profiles

Reply:
The default hadoop version of Spark 3.1 is 3.2. https://mvnrepository.com/artifact/org.apache.spark/spark-core_2.12/3.1.3

Reply:
shall we use 3.2 instead of 2.7